x-pack/test/cases_api_integration/security_and_spaces/tests/trial/cases/observables.ts

```diff
@@ -0,0 +1,212 @@
+import {
+  createCase,
+  deleteAllCaseItems,
+  addObservable,
+  similarCases,
+  updateObservable,
+  deleteObservable,
+  getCase,
+} from '../../../../common/lib/api';
+
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+
+  describe('observables', () => {
+    afterEach(async () => {
+      await deleteAllCaseItems(es);
+    });
+
+    describe('add observable to a case', () => {
+      it('allows the assignees field to be an empty array', async () => {
+        const postedCase = await createCase(supertest, getPostCaseRequest());
+        expect(postedCase.observables).to.eql([]);
+
+        const newObservableData = {
+          value: 'test',
+          typeKey: '0ad4bf8e-575f-49ad-87ea-8bcafd5173e4',
+          description: '',
+        };
+
+        const updatedCase = await addObservable({
+          supertest,
+          caseId: postedCase.id,
+          params: {
+            observable: newObservableData,
+          },
+        });
+
+        expect(updatedCase.observables.length).to.be.greaterThan(0);
+      });
+    });
+
+    describe('shows similar cases', () => {
+      it('returns cases similar to given case', async () => {
+        const [caseA, caseB] = await Promise.all([
+          createCase(supertest, getPostCaseRequest()),
+          createCase(supertest, getPostCaseRequest()),
+          createCase(supertest, getPostCaseRequest()),
+        ]);
+
+        const newObservableData = {
+          value: 'value',
+          typeKey: OBSERVABLE_TYPES_BUILTIN[0].key,
+          description: '',
+        };
+
+        const { cases } = await similarCases({
+          supertest,
+          body: { perPage: 10, page: 1 },
+          caseId: caseA.id,
+        });
+        expect(cases.length).to.be(0);
+
+        await addObservable({
+          supertest,
+          caseId: caseA.id,
+          params: {
+            observable: newObservableData,
+          },
+        });
+
+        await addObservable({
+          supertest,
+          caseId: caseB.id,
+          params: {
+            observable: newObservableData,
+          },
+        });
+
+        const { cases: casesAfterObservablesAreAdded } = await similarCases({
+          supertest,
+          body: { perPage: 10, page: 1 },
+          caseId: caseA.id,
+        });
+
+        expect(casesAfterObservablesAreAdded.length).to.be(1);
+      });
+
+      it('returns cases similar to given case with json in the value', async () => {
+        const [caseA, caseB] = await Promise.all([
+          createCase(supertest, getPostCaseRequest()),
+          createCase(supertest, getPostCaseRequest()),
+          createCase(supertest, getPostCaseRequest()),
+        ]);
+
+        const newObservableData = {
+          value: '{"foo": "bar"}',
```

Comment:
I do not think we should support this kind of values in the MVP. Also `OBSERVABLE_TYPES_BUILTIN[0].key` is an `IPv4`.